### DISCUSSION TITLE
<A-j> Move down & <A-k> Move up In any mode isn't working for me.

### DISCUSSION THREAD

**Discussion by Alexander-Wilkins** [LazyVim/LazyVim#826] [Q&A]
Hello, when I hold down the Alt key and press j or k, in visual mode, or in normal mode, does not move the code line up or down. 

I researched and articles said that <A-j> is holding alt. 

But not sure if I'm ding something incorrect. 

![image](https://github.com/LazyVim/LazyVim/assets/86133577/88ce5f71-402c-4daa-80bd-2497052c85a6)

I'm referencing this part of the documentation https://www.lazyvim.org/keymaps
and also this part of the github page https://github.com/LazyVim/LazyVim/blob/main/lua/lazyvim/config/keymaps.lua

**Not sure how to fix this. Please help! :)** 

If it also helps, i don't have my own keymaps in the configuration directory provided by Lazyvim

**Comment by dpetka2001** [comment]
Do you happen to override these bindings in another part of your configuration? What does `nmap <a-j>` say?

**Reply by Alexander-Wilkins** [reply]
I do not, I only have the default LazyVim configurations.

**Comment by folke** [comment]
Probably your terminal config

**Reply by Alexander-Wilkins** [reply]
Any suggestions where I could locate that in iTerm2? I thumbed through the the profile settings and found this menu, not sure if I should configure this at all? 

![image](https://github.com/LazyVim/LazyVim/assets/86133577/3666c240-4cb7-47a3-ae38-7c734299f7fa)

**Reply by Alexander-Wilkins** [reply]
I wanted to say, thank you for still answering my question! I hope it's not a dumb one! I super appreciate the help! 👍

**Reply by folke** [reply]
I have no idea. I don't use iterm. I just know other people had similar issues and that it was caused by their terminal and/or tmux config.

I also don't use tmux.

**Reply by ghost** [reply]
Use `Esc+` option in iterm2. There is the reason https://stackoverflow.com/a/18925650.

**Comment by davidbk-legit** [comment]
For me, the issue was that Mac generated Greek symbols. I tried this solution, and it worked
[mac-generating](https://stackoverflow.com/questions/60870113/mac-generating-%e2%88%86%cb%9a%c2%ac-characters-instead-of-executing-vscode-shortcuts-that-involve)

**Comment by umutuygar** [comment]
It happened to be **_disabling_** below Aerospace settings for me. Pasting below for the other Aerospace users struggling with this issue.
See: https://nikitabobko.github.io/AeroSpace/commands#focus

.config/aerospace/aerospace.toml
-------------------------------------
alt-h = 'focus left'
alt-j = 'focus down'
alt-k = 'focus up'
alt-l = 'focus right'
